skimage/restoration/deconvolution.py

```diff
@@ -390,3 +391,137 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
         im_deconv[im_deconv < -1] = -1
 
     return im_deconv
+
+
+def blind_richardson_lucy(image, psf=None, iterations=10,
+                          return_iterations=False, clip=False):
+    """Blind Richardson-Lucy deconvolution.
+
+    Parameters
+    ----------
+    image : ndarray
+       Input degraded image (can be N dimensional).
+
+    psf : ndarray, optional
+       A first estimate of the point spread function, same size as image
+
+    iterations : int
```

Comment:
```suggestion
    iterations : int, optional
```

Reply:
iterations is not optional, because you need to specify for how many iterations the algorithm should run